### ISSUE TITLE
[help needed] complete loss of performance after fine-tuning for integration

### ISSUE THREAD

**Issue by xoelmb** [bowang-lab/scGPT#228]
Hi! I'm trying to fine-tune scGPT for integration of multiple libraries. 
After fine-tuning for 5 epochs (recommended is 15 epochs, each takes 8h to run), I have embedded the cells using the saved model and the result looks nothing like the zero-shot embedding, which was quite good, honestly!

First, pictures of what I mean (top is zero-shot, bottom is fine-tuned 5 epochs)
### 1. time variable and same as categorical
![image](https://github.com/user-attachments/assets/888ab7ac-96b1-40c4-a51b-d306dbb63f06)
### 2. annotation
![image](https://github.com/user-attachments/assets/0c2cef53-f137-4062-8176-d9c685af8e26)
### 3. an example of a gene with biological signal
![image](https://github.com/user-attachments/assets/88f15971-5976-44f5-b36d-cf1a6fbbd7e5)

Second, the code of how I loaded my dataset.
```
adata = sc.read_h5ad('../data/combined/adata.h5ad')
ori_batch_col = "library"
adata.obs["celltype"] = adata.obs["celltype_v1"].astype("category")
adata.var = adata.var.set_index("Symbol")
data_is_raw = True
```

The rest of the parameters in the notebook are not modified from the proposed tutorial. 
To embed the data, I had to provide vocab and args JSON files, which I just copied from the pretrained model that I fine tuned.

**Specific questions:**
- My batch variable is the library, and there are >100 different libraries in the dataset.
- Must the celltype and batch variables in the metadata be categorical or it does not matter?
- Which model should we fine-tune? The one pretrained on whole body, or the whole body + continuous training?

Any clue of what's going on with my fine-tuning? There are just too many parameters that I could try to debug in the notebook, so any advice from the authors or the community is deeply appreciated.
